lib3h_protocol/Cargo.toml

```diff
@@ -6,3 +6,5 @@ edition = "2018"
 
 [dependencies]
 holochain_logging = { path = "../logging" }
+serde = "=1.0.89"
+serde_derive = "=1.0.89"
```

Comment:
I don't actually see this crates being used.

Reply:
`#[derive(Serialize, Deserialize` are from `serde_derive`